dynawo/sources/Common/CMakeLists.txt

```diff
@@ -130,6 +130,8 @@ target_include_directories(dynawo_Common
 target_include_directories(dynawo_Common SYSTEM
   PRIVATE
     $<TARGET_PROPERTY:Boost::boost,INTERFACE_INCLUDE_DIRECTORIES>
+  PUBLIC
+    $<TARGET_PROPERTY:Sundials::Sundials_NVECSERIAL,INTERFACE_INCLUDE_DIRECTORIES>
```

Comment:
I would propose to move the matrix to somewhere else as having a dependency on sundials in common is not good in term of software architecture

Reply:
In fact this means that we have a dependency on sundials in the models which is breaking the solver/model separation no?